wger/core/models.py

```diff
@@ -207,6 +222,14 @@ class UserProfile(models.Model):
                        validators=[MinValueValidator(10), MaxValueValidator(100)])
     '''The user's age'''
 
+    birthdate = models.DateField(verbose_name=('Date of Birth'),
+                                 help_text=_('Please use the format: mm/dd/yyyy'),
```

Comment:
Django should be clever enough to adapt the format to the current locale so if browse the application in German you'd enter ``dd.mm.yyyy``. Also if datefields are rendered as date input fields (I'm not 100% sure that they are), the browser will validate the format as well